Update model-lifecycle.mdx (#4351)
scaleway/docs-content: pages/generative-apis/reference-content/model-lifecycle.mdx

As new versions of models are introduced, you have the opportunity to explore them through the Scaleway console.

A model provided through Scaleway Generative APIs may be classified into one of these statuses: `Preview`, `Active`, `Deprecated`, or `End-of-Life` (EOL).

> - **Preview**: This status indicates that the model can be tested but no Service Level Agreements are yet provided. At this stage, there is no guarantee the model will reach `Active` status. In most cases, `Preview` model will still be deployable in dedicated instances using [Managed Inference](https://www.scaleway.com/en/inference/) product.

Reviewer: ```suggestion
- **Preview**: This status indicates that the model can be tested but no service level agreements are yet provided. At this stage, the model is not guaranteed to reach `Active` status. In most cases, `Preview` model will still be deployable in dedicated instances using [Managed Inference](https://www.scaleway.com/en/inference/) product.
```

> - **Active**: This status indicates that the model version is under continuous development, with ongoing updates that may include bug fixes, enhancements and provideService Level Agreemnt

Reviewer 2: ```suggestion
- **Active**: This status indicates that the model version is under continuous development, with ongoing updates that may include bug fixes and enhancements, and provides service-level agreement.
```

- **Deprecated**: A model version is designated deprecated when a newer, more efficient version is available. Scaleway assigns an EOL date to these deprecated versions. Although deprecated versions remain usable, it's recommended to transition to an active version by the EOL date.
- **EOL**: At this stage, the model version is retired and no longer accessible for use. Any attempts to utilize an End-of-Life version will not be successful.

<Message type="note">
  In the Scaleway console, a model version’s status is marked as either **Preview**, **Active** or **Deprecated**. 
</Message>

We guarantee support for new models in `Active` status for at least 8 months starting from their regional launch. Customers will receive a 3-month notice before any model is marked as End-of-Life (EOL).

> We guarantee support for new models in `Preview` status for at least 1 months starting from their regional launch. Customers will receive a 1-month notice before any `Preview` model removed from Generative APIs. 

Reviewer 2: ```suggestion
We guarantee support for new models in `Preview` status for at least 1 month starting from their regional launch. Customers will receive a 1-month notice before any `Preview` model is removed from Generative APIs. 
```

> When removing a model, if an alternative model of a similar type is available in Generative APIs, we may redirect traffic to this alternative model instead of removing the model string from the API. This will prevent any applications not updated in time from breaking completely, although we cannot guarantee models outputs will stay similar.

Reviewer 2: ```suggestion
When removing a model, if an alternative model of a similar type is available in Generative APIs, we may redirect traffic to this alternative model instead of removing the model string from the API. This will prevent applications not updated in time from breaking completely, although we cannot guarantee model outputs will stay similar.
```

> 

Reviewer 2: ```suggestion
```
